### PR TITLE
Modify Contribution file

### LINKED ISSUE #40: contribution file clarify change
#13 
need to clarify whether linking an issue is required.

> ## Review Requirements
- When reviewing, confirm that the code accomplishes the requirements and the acceptance criteria by using one or both ways:
A GitHub issue: the code fixes the problem.
Adding a feature: elaborate in the commit the purpose of the code.
Both cases should have accompanying unit tests that cover all the relevant edge cases.

### PR DESCRIPTION
**Pull request by pazMenachem**
clarification for reviewing requirements.
fix #40

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 #Vagrant
 .vagrant
+
+#idea
+.idea
```
Comment (yftacherzog) [inline .gitignore:135]: What's that?

**Comment by yftacherzog** [comment]
I'm fine with this change. I'd like to see that the rest of the team is ok with the change as it affects everyone's future work.

**Comment by pazMenachem** [comment]
> I'm fine with this change. I'd like to see that the rest of the team is ok with the change as it affects everyone's future work. One comment I do have, is that while the PR asks for PRs to be linked with an issue, it is not linked to an issue itself.

image

thought it link automatically.
will link now.

File: .gitignore (hunk 1 above)
Comment (pazMenachem) [reply .gitignore:135]: i dont know at which stage this file came across.
image
image

Comment (yftacherzog) [reply .gitignore:135]: Does it mean that this change should be on this PR or not?

Comment (pazMenachem) [reply .gitignore:135]: yea why not, added this changed to all my PR if im not mistaken.

Comment (yftacherzog) [reply .gitignore:135]: As for why not, it's because we tend to keep the scope of PRs relevant to what each PR claims to be about.

Few problems with the approach of having it as out-of-scope addition on arbitrary PRs:

- It makes it confusing for code review.
- Suppose that we revert such PR/commit, we will also revert that out of scope change.
- You'll have to rebase all those other PRs once the first one gets merged.

What's the reason for adding it in the first place? Did you add it yourself or was it the IDE that added it? Not including it is not going to have any consequences, unless you accidentally add `.idea` in one of your commits.

Comment (yftacherzog) [reply .gitignore:135]: All in all, if you want it in, just create a small PR that just includes it. Explain there why you want it added, and it can be merged quickly.

Comment (pazMenachem) [reply .gitignore:135]: > All in all, if you want it in, just create a small PR that just includes it. Explain there why you want it added, and it can be merged quickly.

i honestly have no idea how the folder came up, ill just add it in a new PR.

Comment (yftacherzog) [reply .gitignore:135]: Are you using intellij? I'd guess it adds that automatically so you don't push its local configs to your repos.

Comment (pazMenachem) [reply .gitignore:135]: maybe ive opened with pycharm once

Comment (yftacherzog) [reply .gitignore:135]: If you don't need it, then you can also discard that change locally and move on without it.

**Review by Omeramsc — APPROVED** [review]
LGTM

**Review by yftacherzog — APPROVED** [review]

**Review by thetaxas2468 — APPROVED** [review]

**Review by nofaralfasi — APPROVED** [review]
